agent/src/beerocks/monitor/monitor_rssi.cpp

```diff
@@ -337,6 +337,12 @@ void monitor_rssi::process()
                            << ", sta_mac = " << sta_mac << ", dest_ip = " << sta_node->get_ipv4()
                            << ", dst_mac = " << arp_dst_mac;
 
+                if (sta_node->get_ipv4().empty() ||
+                    sta_node->get_ipv4() == network_utils::ZERO_IP_STRING) {
+                    auto ipv4 = network_utils::get_ipv4_from_arp_table(arp_dst_mac);
+                    sta_node->set_ipv4(ipv4);
+                }
```

Comment:
instead of adding this w/a  you should  fix the real problem.
i see that in the past you already added a w/a for this issue https://github.com/prplfoundation/prplMesh/commit/873bf9ed6a3fba6322e8eb001071e991bf108bf1 , but then the `Wait until ipv4 will be received on DHCP event` part was removed .
So now cACTION_CONTROL_CLIENT_START_MONITORING_REQUEST  always sent with  ip=0